### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/WFC3/tvb_flattenramp/TVB_flattenramp_notebook.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#362]
## What
An accessibility scan flagged 31 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/WFC3/tvb_flattenramp/TVB_flattenramp_notebook.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell1">` (selector: `#codecell1`)
- `<pre id="codecell2"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb7q_ima.fits to ./data/mastDownload/HST/ibohbfb7q/ibohbfb7q_ima.fits ...
</pre>` (selector: `#codecell2`)
- `<pre id="codecell4"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb7q_flt.fits to ./data/mastDownload/HST/ibohbfb7q/ibohbfb7q_flt.fits ...
</pre>` (selector: `#codecell4`)
- `<pre id="codecell6"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb7q_raw.fits to ./data/mastDownload/HST/ibohbfb7q/ibohbfb7q_raw.fits ...
</pre>` (selector: `#codecell6`)
- `<pre id="codecell8"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb9q_ima.fits to ./data/mastDownload/HST/ibohbfb9q/ibohbfb9q_ima.fits ...
</pre>` (selector: `#codecell8`)
- `<pre id="codecell10"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb9q_flt.fits to ./data/mastDownload/HST/ibohbfb9q/ibohbfb9q_flt.fits ...
</pre>` (selector: `#codecell10`)
- `<pre id="codecell12"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfb9q_raw.fits to ./data/mastDownload/HST/ibohbfb9q/ibohbfb9q_raw.fits ...
</pre>` (selector: `#codecell12`)
- `<pre id="codecell14"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbdq_ima.fits to ./data/mastDownload/HST/ibohbfbdq/ibohbfbdq_ima.fits ...
</pre>` (selector: `#codecell14`)
- `<pre id="codecell16"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbdq_flt.fits to ./data/mastDownload/HST/ibohbfbdq/ibohbfbdq_flt.fits ...
</pre>` (selector: `#codecell16`)
- `<pre id="codecell18"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbdq_raw.fits to ./data/mastDownload/HST/ibohbfbdq/ibohbfbdq_raw.fits ...
</pre>` (selector: `#codecell18`)
- `<pre id="codecell20"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbgq_ima.fits to ./data/mastDownload/HST/ibohbfbgq/ibohbfbgq_ima.fits ...
</pre>` (selector: `#codecell20`)
- `<pre id="codecell22"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbgq_flt.fits to ./data/mastDownload/HST/ibohbfbgq/ibohbfbgq_flt.fits ...
</pre>` (selector: `#codecell22`)
- `<pre id="codecell24"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbgq_raw.fits to ./data/mastDownload/HST/ibohbfbgq/ibohbfbgq_raw.fits ...
</pre>` (selector: `#codecell24`)
- `<pre id="codecell26"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbkq_ima.fits to ./data/mastDownload/HST/ibohbfbkq/ibohbfbkq_ima.fits ...
</pre>` (selector: `#codecell26`)
- `<pre id="codecell28"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbkq_flt.fits to ./data/mastDownload/HST/ibohbfbkq/ibohbfbkq_flt.fits ...
</pre>` (selector: `#codecell28`)
- `<pre id="codecell30"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbkq_raw.fits to ./data/mastDownload/HST/ibohbfbkq/ibohbfbkq_raw.fits ...
</pre>` (selector: `#codecell30`)
- `<pre id="codecell32"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbpq_ima.fits to ./data/mastDownload/HST/ibohbfbpq/ibohbfbpq_ima.fits ...
</pre>` (selector: `#codecell32`)
- `<pre id="codecell34"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbpq_flt.fits to ./data/mastDownload/HST/ibohbfbpq/ibohbfbpq_flt.fits ...
</pre>` (selector: `#codecell34`)
- `<pre id="codecell36"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbfbpq_raw.fits to ./data/mastDownload/HST/ibohbfbpq/ibohbfbpq_raw.fits ...
</pre>` (selector: `#codecell36`)
- `<pre id="codecell38"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/ibohbf040_asn.fits to ./data/mastDownload/HST/ibohbf040/ibohbf040_asn.fits ...
</pre>` (selector: `#codecell38`)
- `<table id="table140572639306576" class="table-striped table-bordered table-condensed">` (selector: `#table140572639306576`)
- `<pre id="codecell45">` (selector: `#codecell45`)
- `<pre id="codecell68">` (selector: `#codecell68`)
- `<pre id="codecell69">` (selector: `#codecell69`)
- `<pre id="codecell73">` (selector: `#codecell73`)
- `<pre id="codecell81">` (selector: `#codecell81`)
- `<pre id="codecell84">` (selector: `#codecell84`)
- `<pre id="codecell89">` (selector: `#codecell89`)
- `<pre id="codecell96">` (selector: `#codecell96`)
- `<pre id="codecell98">` (selector: `#codecell98`)
- `<pre id="codecell99">` (selector: `#codecell99`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
